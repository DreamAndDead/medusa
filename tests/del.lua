local v = "exist"
assert(v)
v = nil
return {
    v = v,
}
