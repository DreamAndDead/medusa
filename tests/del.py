a, b, c = 1, 2, 3
print(a, b, c)
del a, b
print(a, b, c)
del c
print(a, b, c)