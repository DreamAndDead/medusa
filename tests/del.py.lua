dofile('./polyfill/pypolyfill.lua')
local v = "exist"
assert(v)
v = nil
