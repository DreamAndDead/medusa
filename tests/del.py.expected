1	2	3
nil	nil	3
nil	nil	nil
